config/tcs_config.toml

```diff
@@ -50,11 +51,13 @@ ias_url = "https://api.trustedservices.intel.com/sgx/dev"
 # Proxy for https. Leave commented out for direct Internet connections or
 # uncomment and change to your corporate proxy.
 #https_proxy = "http://your-proxy:your-port/"
+https_proxy = "http://proxy-chain.intel.com:912"
 
 # IAS API key is a 32-digit hex string subscription key used for authentication
 # of requests submitted to the IAS server. Obtain the key by subscribing in
 # the portal https://api.portal.trustedservices.intel.com/
-ias_api_key = "112233445566778899aabbccddeeff00"
+#ias_api_key = "112233445566778899aabbccddeeff00"
+ias_api_key = "7e3d31484faa4c87b5d62006f4619343"
```

Comment:
remove the unwanted commented code. and why there is change in ias_api_key?

Reply:
Do not use a real IAS API key here.  Keep the original example.

Reply:
I will keep the original dummy value and delete real IAS api key